game/src/character/objects/TreasureChest.java

```diff
@@ -0,0 +1,85 @@
+package character.objects;
+
+import basiselements.AnimatableElement;
+import character.player.Hero;
+import collision.CharacterDirection;
+import collision.Collidable;
+import collision.Hitbox;
+import graphic.Animation;
+import java.util.ArrayList;
+import java.util.List;
+import level.elements.ILevel;
+import textures.TextureHandler;
+import tools.Point;
+
+public class TreasureChest extends AnimatableElement implements Collidable {
+    private Animation closed;
+    private Animation opening;
+    private Animation opened;
+    private Animation currentAnimation;
+    private Point currentPosition;
+    private ILevel currentLevel;
+    private boolean isOpen = false;
+    private int counter = 15;
```

Comment:
`counter` in `openingAnimationTime` umbenennen, das macht es einfacher zu lesen.